web-ui/controller/api/git_forge.ml

```diff
@@ -0,0 +1,82 @@
+module type Api = Api_view.Git_forge.Api
+
+module Client = Ocaml_ci_api.Client
```

Comment:
```suggestion
```
This can be removed as you bind the name inside `module Make` later on. You could even move the `module Run_time` into that module.